tests/Integration/Behaviour/Features/Scenario/Search/add_alias.feature

```diff
@@ -0,0 +1,47 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s alias --tags add
+@restore-aliases-before-feature
+@clear-cache-before-feature
+@clear-cache-after-feature
+@add
+Feature: Add basic alias from Back Office (BO)
+  As a BO user
+  I need to be able to add new alias with basic information from the BO
+
+  Scenario: I add an alias with basic information
+    Given following aliases should exist:
+      | id reference | alias  | search |
+      | alias1       | bloose | blouse |
+      | alias2       | blues  | blouse |
+    When I add alias with following information:
+      | alias   | large |
+      | search  | big   |
+    Then following aliases should exist:
+      | id reference | alias  | search |
+      | alias1       | bloose | blouse |
+      | alias2       | blues  | blouse |
+      | alias3       | large  | big    |
+
+  Scenario: I add a multiple aliases with basic information
+    When I add alias with following information:
+      | alias   | bliu,blu |
+      | search  | blue     |
+    Then following aliases should exist:
+      | id reference | alias  | search |
+      | alias1       | bloose | blouse |
+      | alias2       | blues  | blouse |
+      | alias3       | large  | big    |
+      | alias5       | bliu   | blue   |
+      | alias6       | blu    | blue   |
+
+  Scenario: I add a multiple aliases with duplicate search field
+    When I add alias with following information:
```

Comment:
Before this step you need to assert that the alias with search "blue" exists to validate your scenario (the aliases exists from previous scenario I guess? which is not ideal should suffice. Anyway, you need to add at least this first:
```
    Given following aliases should exist:
      | id reference | alias  | search |
      | alias1       | bloose | blouse |
      | alias2       | blues  | blouse |
      | alias3       | large  | big    |
      | alias5       | bliu   | blue   |
      | alias6       | blu    | blue   |
```

Reply:
I always forget what we decide with the behavior (cuz there are no written specs :smile:), but seeing the scenario - the final behavior is quite primitive - so in case the searchTerm+alias combination already exists we just skip it and for all new ones we create new alias entities. (so we don't throw any exception and we don't "magically" delete all the other aliases for that search term). That should work well for creation.
For edition we will probably need abit more complicated solution. (but its for next PR.). Just some scenarios to consider when we implement **edition**: 
1. Editing aliases `without changing the search term`. When opening the edition form all the aliases will be prefilled so in this case it probably makes sense replacing all the aliases - so it will delete/create new entities.
2. Editing aliases when changing the search term and the `search term doesn't exist yet`. This one is easy, we just create entities for all the aliaseses typed in the input.
3. Editing aliases when changing the search term and the `search term already exists`. This one probably most complicated because if we don't prefill the `aliases` input from the existing search term dynamically, then it will introduce that strange behavior ending up deleting multiple alias entities accidentally. So for this case, the easiest solution would probably be just to throw exception every time the searchTerm exists (and this means shouldn't probably even need to use select2 component and just leave the input free text as it is in legacy page)

Reply:
Yes, though these scenarios will need to be implemented with other PR's that has delete/edit commands I suppose  :)